pyfolio/plotting.py

```diff
@@ -184,7 +184,7 @@ def plot_rolling_fama_french(returns,
 
     ax.set_ylabel('Beta')
 
-    rolling_beta = timeseries.rolling_fama_french(
+    rolling_beta = timeseries.rolling_regression(
```

Comment:
This should also be `rolling_correlation` not `rolling_regression` - missed that.

Reply:
Good catch, thanks.

Reply:
Actually should be `rolling_regression` throughout.